server/src/main/java/com/linecorp/centraldogma/server/internal/api/HttpApiExceptionHandler.java

```diff
@@ -62,6 +63,9 @@
                .put(EntryNotFoundException.class,
                     (ctx, req, cause) -> newResponse(ctx, HttpStatus.NOT_FOUND, cause,
                                                      "Entry '%s' does not exist.", cause.getMessage()))
+               .put(EntryNoContentException.class,
+                    (ctx, req, cause) -> newResponse(ctx, HttpStatus.NO_CONTENT, cause,
+                                                     "Entry '%s' does not have content.", cause.getMessage()))
```

Comment:
`204 No Content` response must have no content, which means you do not need to prepare any message.

Reply:
Thank you for review. Fixed in 7886e0d.